patches/@privy-io+react-auth.readme.md

```
@@ -22,8 +22,18 @@ Embedded wallet is only available over HTTPS
-Embedded wallet is only available over HTTPS/CHROME-EXTENSION/FILE
+Privy Embedded wallet is only available over HTTPS/CHROME-EXTENSION/FILE
 ```
 
 
+## Domain whitelist
 
+```js
+["localhost","127.0.0.1"].indexOf(window.location.hostname)
+```
+
+to 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)